src/components/ellipse/Ellipse.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { DefaultShapeProps, CornerProps } from '../../types';
+import {
+    LayoutStyleProperties,
+    transformLayoutStyleProperties
+} from '../../styleTransformers/transformLayoutStyleProperties';
+import { useYogaLayout } from '../../hooks/useYogaLayout';
+import { transformBlendProperties, BlendStyleProperties } from '../../styleTransformers/transformBlendProperties';
+import {
+    GeometryStyleProperties,
+    transformGeometryStyleProperties
+} from '../../styleTransformers/transformGeometryStyleProperties';
+import { useFillsPreprocessor } from '../../hooks/useFillsPreprocessor';
+
+export interface EllipseNodeProps extends DefaultShapeProps, CornerProps {
```

Comment:
I suggest calling it `EllipseProps` like any other interfaces (e.g. `TextProps`, `GroupProps`, etc)

Reply:
Sure. I agree with you.